README.md

```diff
@@ -45,6 +45,11 @@ Use images and backgrounds to enhance the look of your website.
   </ul>
 </details>
 
+<!-- LIVE DEMO -->
+
+## 😎 Live Demo <a name="live-demo"></a>
+Click here to see a live demo of this project online: <li><a href="https://joyapisi.github.io/MyPortfolio/">Git Hub</a></li>
+
 
```

Comment:
- [ ] - [ OPTIONAL ] I have some optional changes I think you can work on later to make your app even better, let's start with this name on line 51, it is a bit confusing to it is the link to your online version( deployed link) but you have written `GitHub`, you should rather call it live demo, website, or portfolio but please not GitHub.
- [ ] - [OPTIONAL ] In your mobile and desktop versions, we have some broken adjustments as shown in the image below, please make sure you fix that. take a look back to the Figma design to fix these issues

<img width="372" alt="2" src="https://user-images.githubusercontent.com/77843050/208408256-2ccbe556-cd57-4136-b204-9fa4e0f187f6.png">

<img width="381" alt="1" src="https://user-images.githubusercontent.com/77843050/208408241-da4f232e-5901-46d9-9412-3b89e6a382f1.png">


<img width="1279" alt="3" src="https://user-images.githubusercontent.com/77843050/208408524-66a69bd0-5c65-4f8a-8cdb-2366d7fd46c7.png">

Reply:
Thank you for pointing these out. I will definitely work on the alignment issues.